test/tests/rackhd11/test_rackhd11_api_templates.py

```diff
@@ -26,17 +29,13 @@ def test_api_11_templates_library(self):
                     print "Checking:", item['name'], subitem
                 self.assertGreater(len(item[subitem]), 0, subitem + ' field error')
 
-    def test_api_11_templates_library_ID(self):
-        api_data = fit_common.rackhdapi("/api/1.1/templates/library")
-        self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
-        for item in api_data['json']:
-            lib_data = fit_common.rackhdapi("/api/1.1/templates/library/" + item['name'])
-            self.assertEqual(lib_data['status'], 200, "Was expecting code 200. Got " + str(lib_data['status']))
-            # check required fields
-            for subitem in ['contents', 'createdAt', 'id', 'name', 'updatedAt']:
-                if fit_common.VERBOSITY >= 2:
-                    print "Checking:", item['name'], subitem
-                self.assertGreater(len(item[subitem]), 0, subitem + ' field error')
+    def test_api_11_templates_library_ID_put_get_delete(self):
+            api_data = fit_common.rackhdapi("/api/1.1/templates/library/testid?scope=global", action="text-put", payload="null")
+            self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
```

Comment:
Double-checking.  In the 2.0 api tests it checks for status code of 201.  Is the behavior different in 1.1 vs. 2.0?